modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc

```diff
@@ -0,0 +1,87 @@
+[id="installing-image-puller-using-che-operator_{context}"]
+= Installing {image-puller-name-short} using the {prod-short} Operator
+
+This section describes how to use the {prod-short} Operator to install the {image-puller-name-short}. This is a Community-Supported tech preview feature.
+
+.Prerequisites
+
+* xref:defining-the-list-of-images-to-pull.adoc[]
+* xref:defining-the-memory-parameters-for-the-image-puller.adoc[]
+ifeval::["{project-context}" == "che"]
+* Operator Lifecycle Manager is available on the {kubernetes} instance.
+endif::[]
+* OperatorHub is available on the OpenShift instance.
```

Comment:
@themr0c I think you did a [valid comment](https://github.com/eclipse/che-docs/pull/1800/files#r566695500) on the original PR. I think that this prerequisite makes sense if {prod-short} is CRW only.

@ibuziuk can you confirm?

Reply:
well, `OperatorHub is available on the OpenShift instance.` I think this is always true, right? So, we can skip this line at all IMO

Reply:
Fine